CHANGELOG.md

```diff
@@ -2,8 +2,6 @@
 
 ## 2.6.4 - July 20, 2024
 
-Add support for [Rails 7.2](https://github.com/rails/rails/releases/tag/v7.2.0)
```

Comment:
?

Reply:
the 3 PRs listed below was about supporting changes in 7.2, thus that title here

Reply:
Ah, 2.6.4 didn't actually support 7.2. That combined with seeing [this comment](https://github.com/rubysherpas/paranoia/pull/561#issuecomment-2283237082) made me think this line was added by mistake. So I removed it as part of this PR.

Reply:
make sense, I should have been more clear about it. I could not bump the version until it was officially released. but those were preliminary fix that was shown to be required for those using the pre-release.